src/ci/docker-compose/cromwell-test/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM openjdk:8
+FROM us.gcr.io/broad-dsp-gcr-public/base/jre:8-debian
+
+RUN apt-get update -q && \
+    apt-get upgrade -qq && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
```suggestion
FROM us.gcr.io/broad-dsp-gcr-public/base/jre:8-debian
```

Reply:
Yep you're right!

Reply:
They do indeed already get run when the image is constructed but I don't think there's any downside to re-running at our own image build time as well?

 Remember that we're also re-running Trivy against the new images we build so we aren't "losing our green tick" and might possibly bring in important new updates that way (eg if the "official" images are only pushed on an infrequent schedule

Reply:
a.k.a. DRY

Reply:
I agree - and we update those images weekly. We can also switch to a daily schedule if you think that'd be helpful. Though if that's not good enough, then please feel free to just use the `adoptopenjdk:8` image - that's as good as our "blessed" ones, but we just wanted to simply that part of work for you 😉

Reply:
🤷‍♂️  I don't feel particularly strongly so I'll switch it over to "just use the base image"